search/tests/test_api.py

```diff
@@ -0,0 +1,31 @@
+"""
+Tests for search API functions.
+"""
+
+from __future__ import unicode_literals
+
+from importer.tasks import import_file
+from learningresources.api import create_repo
+from search.api import construct_queryset
+from search.tests.base import SearchTestCase
+
+
+class TestAPI(SearchTestCase):
+    """Test API."""
+
+    def import_course_tarball(self, repo):
+        """
+        Import course into repository.
+        """
+        tarball_file = self.get_course_single_tarball()
+        import_file(tarball_file, repo.id, self.user.id)
+
+    def test_repository_exact_match(self):
+        """
+        Test that repository matches exactly
+        """
+        test_repo = create_repo("test", "testing", self.user.id)
+        testing_repo = create_repo("testing", "testing", self.user.id)
+        self.import_course_tarball(test_repo)
+
+        self.assertEqual(construct_queryset(testing_repo.slug).count(), 0)
```

Comment:
I'd like the test to assert that both contain zero originally, then afterward that one has stuff and the other is still zero.